### DISCUSSION TITLE
Multiple Plants

### DISCUSSION THREAD

**Discussion by RobbieDemaegdt** [djansen1987/SAJeSolar#29] [Ideas]
Hello,

I have installed the integration but I see that I have errors in the logging because I have the H1 sensors active in my configuration but apparently the python code is hard coded to use the first plant.
But my H1 sensors are in my second plant.
Would it be a good idea to have an extra configuration parameter input to say which plant to use (default: 0 for backwards compatibility)
I would say I do it myself but I have no experience in creating integrations for home assistant and my python code is also not up to date anymore.

**Comment by djansen1987** [comment]
Good idea, it is hard to do as i do not have a plant login with multiple plants to develop the code for it. I am open to the idea for it.

**Reply by djansen1987** [reply]
i see you also opened the pull request, thanks for that!
